lib/vyasa_web/live/display_manager/display_live.ex

```diff
@@ -505,6 +540,15 @@ defmodule VyasaWeb.DisplayManager.DisplayLive do
     socket
   end
 
+  defp mutate_verses(%Socket{} = socket, target_verse_id, mutated_verses) do
```

Comment:
will be good to add a simple doc string since this stream insertion behaviour was a hard gotcha right

Reply:
merging this in first, add docstring sometime soon yeah